<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_descuentos_aplicados">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>650</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 12pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Agregar nuevo Laboratorio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>55</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(111, 111, 111);
font: bold 7pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Añade información del laboratorio.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_6">
          <property name="minimumSize">
           <size>
            <width>225</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Agregar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_agNombreMedicamento">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_12">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>60</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Editar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_10">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QComboBox" name="cbox_agPresentacion_2">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QComboBox {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_9">
          <property name="minimumSize">
           <size>
            <width>225</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Eliminar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>15</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>InterfaceImages/lista-de-verificacion (1).png</normaloff>InterfaceImages/lista-de-verificacion (1).png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_9">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
             </property>
             <property name="text">
              <string>Actualizar</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>InterfaceImages/advertencia.png</normaloff>InterfaceImages/advertencia.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
             </property>
             <property name="text">
              <string>Eliminar</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>InterfaceImages/eliminar.png</normaloff>InterfaceImages/eliminar.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QFrame" name="frame_5">
          <property name="maximumSize">
           <size>
            <width>650</width>
            <height>75</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 10pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>InterfaceImages/advertencia.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 10pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>¡Alerta de seguridad!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: bold 9pt &quot;Nunito&quot;;</string>
             </property>
             <property name="text">
              <string>Contraseña:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background-color: #f0f0f0; /* Color de fondo deseado */
    border: 1px solid #ccc; /* Color del borde deseado */
    padding: 5px; /* Espacio alrededor del texto */
    color: rgb(0, 0, 0); /* Color del texto deseado */
  	font: 9pt &quot;Nunito&quot;; /* Fuente deseada */
    font-size: 14px; /* Tamaño de fuente deseado */
    border-radius: 8px; /* Redondear las esquinas */
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>75</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_ingresar">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 1px solid #ccc;
    border-radius: 10px;           /* Bordes redondeados con radio de 10 píxeles */
    padding: 5px;
	font: bold 9pt &quot;Nunito&quot;;                  /* Espaciado interno del botón */
}

QPushButton:hover{
font: bold 9pt &quot;Nunito&quot;;
background-color: #f0f0f0; /* Color de fondo deseado */
}
</string>
             </property>
             <property name="text">
              <string>Ingresar</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>InterfaceImages/lista-de-verificacion (1).png</normaloff>InterfaceImages/lista-de-verificacion (1).png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
